streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -317,7 +317,7 @@ public void prepareCommit() {
                 stateMgr.flush();
                 recordCollector.flush();
 
-                log.info("Prepared task for committing");
+                log.debug("Prepared task for committing");
```

Comment:
These were absolutely spamming the logs, and don't seem like general-interest "info", so I demoted them to debug. But if they were set to info for a good reason I'm happy to revert

Reply:
I think at TaskManager we already log at INFO level for time-based and user-based committing, so I'm fine with this.